js/app.js

```diff
@@ -211,7 +214,7 @@
 					} else if (fullscreenElem.msRequestFullscreen) {
 						fullscreenElem.msRequestFullscreen();
 					}
-					$(this).attr('data-original-title', 'Exit fullscreen');
+					$(this).attr('data-original-title', t('spreed', 'Exit fullscreen'));
```

Comment:
Please don't change unrelated parts of the file, this should be in a separate PR (applies to other parts below, too).